src/modules/overlay/module.spec.js

```diff
@@ -0,0 +1,263 @@
+/* eslint-disable max-nested-callbacks */
```

Comment:
You could disable `max-nested-callbacks` in the eslint config for all `*.spec*` files with the `oerrides` option:
https://eslint.org/docs/user-guide/configuring#disabling-rules-only-for-a-group-of-files

Reply:
ah wow cool I didn't know it